xc/xenops_server_xen.ml

```diff
@@ -741,6 +772,10 @@ module HOST = struct
          let features_pv = get_cpu_featureset xc Featureset_pv in
          let features_hvm = get_cpu_featureset xc Featureset_hvm in
          let features_oldstyle = oldstyle_featuremask xc in
+         let policy_hvm = serialise (Xenctrlext.xc_cpu_policy_get_system xc
+             (Hashtbl.find xen_sysctl_cpu_policy "XEN_SYSCTL_cpu_policy_hvm_default")) in
+         let policy_pv = serialise (Xenctrlext.xc_cpu_policy_get_system xc
+             (Hashtbl.find xen_sysctl_cpu_policy "XEN_SYSCTL_cpu_policy_pv_default")) in
```

Comment:
`Hashtbl.find` can fail - the exception should be caught at least at some outer level.